src/wiktextract/extractor/en/page.py

```diff
@@ -1630,6 +1656,16 @@ def sense_template_fn(
                 return None
             if is_panel_template(wxr, name):
                 return ""
+            if name in info_template_funcs:
+                info_data, info_exp = parse_info_template_arguments(
+                    wxr, name, ht, "sense"
+                )
+                if info_data or info_exp:
+                    if info_data:
+                        data_append(sense_base, "info_templates", info_data)
+                    if info_exp and isinstance(info_exp, str):
+                        return info_exp
+                    return ""
```

Comment:
Should use `parse_info_template_node()` directly on `contents` nodes, I guess `+obj` template's texts are only supposed to be removed from header line texts but not gloss texts. And we could also avoid recreating the template wikitext string.

Reply:
I left the `+obj` in the gloss because the gloss might not make sense otherwise. I don't understand the rest of the comment.

Reply:
I mean the code looks like it want to change the expanded text of some templates but the expanded text is not changed. And `parse_info_template_arguments()` also rebuilds the template wikitext from it's arguments, which feels it could go wrong if not carefully consider many cases, process the original `TemplateNode` object seems more reliable.

Reply:
This may be premature, but if someone wants to change the expanded text in the future... But it's true it currently doesn't do anything, and might never do anything, and all of this could just remain a kludge for `+obj`.

Doing the template parsing in sense really seems the most natural in `template_fn=` (in this case `sense_template_fn`), because that's where all the other templates are handled, and it only has access to `name` and `ht`. I'm not really happy about it either, if you can come up with something I'm all ears.